sdk/cosmosdb/Microsoft.Azure.Management.CosmosDB/src/Generated/Models/RestorableMongodbDatabaseGetResult.cs

```diff
@@ -0,0 +1,63 @@
+// <auto-generated>
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// </auto-generated>
+
+namespace Microsoft.Azure.Management.CosmosDB.Models
+{
+    using Microsoft.Rest;
+    using Microsoft.Rest.Serialization;
+    using Newtonsoft.Json;
+    using System.Collections;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    /// <summary>
+    /// An Azure Cosmos DB restorable MongoDB database
+    /// </summary>
+    [Rest.Serialization.JsonTransformation]
+    public partial class RestorableMongodbDatabaseGetResult : ARMResourceProperties
+    {
+        /// <summary>
+        /// Initializes a new instance of the
+        /// RestorableMongodbDatabaseGetResult class.
+        /// </summary>
+        public RestorableMongodbDatabaseGetResult()
+        {
+            CustomInit();
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the
+        /// RestorableMongodbDatabaseGetResult class.
+        /// </summary>
+        /// <param name="id">The unique resource identifier of the ARM
+        /// resource.</param>
+        /// <param name="name">The name of the ARM resource.</param>
+        /// <param name="type">The type of Azure resource.</param>
+        /// <param name="location">The location of the resource group to which
+        /// the resource belongs.</param>
+        public RestorableMongodbDatabaseGetResult(string id = default(string), string name = default(string), string type = default(string), string location = default(string), IDictionary<string, string> tags = default(IDictionary<string, string>), ManagedServiceIdentity identity = default(ManagedServiceIdentity), RestorableMongodbDatabasePropertiesResource resource = default(RestorableMongodbDatabasePropertiesResource))
```

Comment:
@vinhms can you check if this is true? i.e. at our service side do we have each of location, tags and ManagedServiceIdentity for RestorableMongodbDatabaseGetResult. 
If not, then we should fix this in swagger.